### DISCUSSION TITLE
Close chrome when exiting game.

### DISCUSSION THREAD

**Discussion by ASweed** [ChristopherKlay/StadiaEnhanced#244] [Ideas & Requests]
I use a lot of click to play shortcuts to the games and would love if it is possible that when I hold escape and then press "close game" it also closes the chrome window instead of going back to the Stadia home screen.

**Comment by ChristopherKlay** [comment]
Due to security reasons, there's limitations (i.e. scripts only being able to close windows created by scripts — not the user) when it comes to window handling that don't allow this.
